### PR TITLE
Feature/local prices

### PR DESCRIPTION
**Pull request by andrew-skrypnyk**
local prices support
deeplinks support
appstore promo products support

### CODE REVIEW

File: PandaSDK.podspec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |spec|
   #
 
   spec.name         = "PandaSDK"
-  spec.version      = "0.0.8"
+  spec.version      = "0.1.0"
```
Comment (InnaLabynskaya) [inline PandaSDK.podspec:19]: fix version - it is wrong

File: Sources/PandaSDK/ConfiguredPanda.swift [hunk 2]
```diff
@@ -218,6 +231,13 @@ final public class Panda: PandaProtocol {
         return controller
     }
     
+    private func presentOnRoot(`with` viewController: UIViewController, _ completition: (() -> Void)? = nil) {
```
Comment (InnaLabynskaya) [inline Sources/PandaSDK/ConfiguredPanda.swift:234]: completition typo

File: Sources/PandaSDK/Extension+StoreKit.swift [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+        default:
+            break
+        }
+        
+        let unit = unitStringFrom(periodUnit: discount.subscriptionPeriod.unit)
+        
+        return ["unit" : unit, "units_count" : unit_count, "periods_count" : periods_count, "mode" : mode, "price" : discount.price.floatValue, "offer_id" : discount.identifier ?? ""]
+    }
+    
+    @available(iOS 12.2, *)
```
Comment (InnaLabynskaya) [inline Sources/PandaSDK/Extension+StoreKit.swift:73]: but in specification min version is 12, maybe we can update version in spec?

File: Sources/PandaSDK/Extension+StoreKit.swift [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+extension SKProduct {
+    
+    func localizedString(with offerID : String? = nil) -> String {
+        
+        guard let offer = offerID else {
+            return localizedPrice()
+        }
+        
+        guard #available(iOS 12.2, *) else {
```
Comment (InnaLabynskaya) [inline Sources/PandaSDK/Extension+StoreKit.swift:19]: Let's update version in spec

File: Sources/PandaSDK/Extension+StoreKit.swift [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+                mode = "pay_as_you_go"
+            case .freeTrial:
+                mode = "trial"
+            default:
+                break
+            }
+            
+            let intro_unit = unitStringFrom(periodUnit: intro.subscriptionPeriod.unit)
+            
+            if let aMode = mode{
```
Comment (InnaLabynskaya) [inline Sources/PandaSDK/Extension+StoreKit.swift:105]: aMode - weird naming + missing spacing

File: PandaSDK.podspec (hunk 1 above)
Comment (andrew-skrypnyk) [reply PandaSDK.podspec:19]: why? yours was `0.0.9`, then should be `0.1.0`.
or `0.0.10`?

File: Sources/PandaSDK/Extension+StoreKit.swift (hunk 3 above)
Comment (andrew-skrypnyk) [reply Sources/PandaSDK/Extension+StoreKit.swift:73]: we still should use 11.2 as min spec version ;(

File: Sources/PandaSDK/Extension+StoreKit.swift (hunk 4 above)
Comment (andrew-skrypnyk) [reply Sources/PandaSDK/Extension+StoreKit.swift:19]: same

File: PandaSDK.podspec (hunk 1 above)
Comment (andrew-skrypnyk) [reply PandaSDK.podspec:19]: will be `0.0.9`

**Comment by andrew-skrypnyk** [comment]
🥳

**Pull request merged**